src/components/DroughtThresholds.vue

```diff
@@ -1292,6 +1296,16 @@
             <use xlink:href="#shadeMask-d"/>
             <use xlink:href="#healthy_sunflower"/>
             <use xlink:href="#woodcut_texture"/>
+            <g class="streamtext">
+              <text transform="matrix(1 0 0 1 24 55)">Normal</text>
```

Comment:
GC: The font size on the chart is looking a little small on my laptop screen. I think any adjustments can wait until FSP feedback is in, but something to consider adjusting.

Reply:
Sounds good, I also think it's on the small side, but agree, we can adjust after/with final IPDS edits.